support-frontend/assets/helpers/page/page.js

```diff
@@ -134,11 +132,23 @@ function init<S, A>(
   pageReducer?: CommonState => Reducer<S, A> | null,
   thunk?: boolean = false,
 ): Store<*, *, *> {
-
   try {
+    /**
+     * Dynamically load @guardian/consent-management-platform
+     * on condition we're not server side rendering (ssr) the page.
+     * @guardian/consent-management-platform breaks ssr otherwise.
+     */
+    if (!getGlobal('ssr') && ccpaEnabled()) {
```

Comment:
Could we check the user's country here to avoid loading the cmp when we don't need to?

Reply:
That's what's happening, we only import `@guardian/consent-management-platform` when `ccpaEnabled()` is `true` (if the user is in the US).

Reply:
Ah I've just seen that `ccpaEnabled` does check that 👍